### ISSUE TITLE
add randomly generated seed to study metadata so each phantom can be reproduced

### ISSUE THREAD

**Issue by brandonjnelsonFDA** [bnel1201/PedSilicoICH#64]
Already built into run_study: https://github.com/brandonjnelsonFDA/PedSilicoICH/blob/3972e48a9944aded7ec41b618cef1afd26b36053/src/pedsilicoICH/pipeline.py#L178

**Comment by brandonjnelsonFDA** [comment]
Close but not quite there:

![Image](https://github.com/user-attachments/assets/ace9ec3d-7a83-4b79-92c2-87ded0f48397)


Random seeds are assigned for each case, but this makes the round lesions perfectly spherical (removes their randomness), which isnt desired)

![Image](https://github.com/user-attachments/assets/1d5d6e67-2d18-4c4d-96e2-72d78dbaee9a)

**Comment by brandonjnelsonFDA** [comment]
completed with small fix to master in [e435247](https://github.com/DIDSR/PedSilicoICH/commit/e435247590cab42baaa64465cadfceaedc63da62)

```python
from pedsilicoICH.ground_truth_definition.phantoms import load_phantom
from notebooks.utils import ctshow
import matplotlib.pyplot as plt

for _ in range(2):
    phantom = load_phantom(6.5)
    phantom.insert_lesion('round', seed=42, complexity=3)
    ctshow(phantom.get_CT_number_phantom()[phantom._lesion_coords[0][0]])
    plt.show()
```

yields complex, yet repeatable round lesions:

![image](https://github.com/user-attachments/assets/aeab027c-d609-4a07-beff-e72777cd0c68)
